README.md

```diff
@@ -20,6 +20,17 @@ Skapa en `env.json`-fil med följande för att få stripe-integrationen att fung
 }
 ```
 
+För att installera alla appens beroenden körs kommandot:
+```bash
```

Comment:
Bash seems a bit excessive, it works in any shell right? Then it should just be `sh`. I know it's a small thing, but only if it actually has to be in bash then it should be marked as such (or rewritten to be portable).

Reply:
True! ^^